lib/perl/Genome/Model/Tools/Analysis/DumpIgvXmlMulti.pm

```diff
@@ -7,6 +7,7 @@ use Genome;
 use Command;
 use IO::File;
 use Cwd qw(abs_path);
+use URI::URL;
```

Comment:
I am giving in to sharing a personal preference... maybe even a rant...

It really bugs me that Perl modules export things by default.  I would much rather see things imported explicitly (and not imported if unused).  I make certain exceptions, e.g. `Test::More`, but they are very few.  For example, `URI::URL` exports `url()` by default.  Even if `url()` was being used it would not necessarily be obvious where it came from (more so with more generic method names like, `move`).

I try to do,

```
use URI::URL qw();
```

The `qw()` is intentional as opposed to, for example, `""`.  The reason is that an empty list is different than passing an empty string through and I think empty list is most correct.

Another alternative is,

```
require URI::URL;
```

However, in addition to not importing default exported methods it doesn't call `import()` at all and delays loading the module until runtime.  There are definitely cases where delaying loading until runtime is desired; it's maybe even a better default for performance reasons.  I don't know that not calling `import()` at all causes problems.

So I don't know whether `use Foo qw()` is a better default practice than `require Foo` but in my opinion either is better than `use Foo`.

P.S. maybe we should also, or maybe alternatively, use `namespace::autoclean`.